libnavui-maps/src/main/res/drawable/mapbox_bg_button.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#ffffff" />
```

Comment:
Instead of using hardcoded colors, can we use them from `colors.xml`? Same applies to all the other colors in this entire PR